docs/source/courses/collaborative-coding.md

```diff
@@ -17,10 +17,10 @@ Students should have some initial experience with both Python and version contro
 Students should bring their own laptop with Python installed. If you require any assistance, please contact
 <a href="mailto:adam.tyson@ucl.ac.uk?subject=SWC/GCNU Software Skills">Adam Tyson</a> in advance of the course.
 
-They should also follow [these setup instructions](https://github.com/rse-best-practices-course-2023/rse-best-practices-playground/issues/1) in advance of the course.
+They should also follow [these setup instructions](https://github.com/rse-best-practices-course/rse-best-practices-playground/issues/1) in advance of the course.
```

Comment:
The problem now (maybe to be tackled separately), is that we might want to include the date in the exercise repo, a this now points to an archived one?

```suggestion
They should also follow [these setup instructions](https://github.com/rse-best-practices-course/rse-best-practices-playground-2024/issues/1) in advance of the course.
```

Reply:
Or we entirely delete the archived repo from last year (but the student's might use it as a reference, and their forks will continue to exist?)

Reply:
Actually, this made me think that we do not want them to follow those setup instructions anymore, so actually this reference is outdated now 😱 shall we delete it directly?

Reply:
Yea, I think so. We should mention that they need Git and Github setup too? We can link them to https://neuroinformatics.dev/slides-SWC-PhD-intro/#/section

Reply:
```suggestion
They should also follow [these setup instructions](https://neuroinformatics.dev/slides-SWC-PhD-intro/#/section) in advance of the course.
```

Reply:
If this is OK to you I commit my latest suggestion and merge

Reply:
great! Yes, this is OK with me!